core/src/mill/define/Task.scala

```diff
@@ -93,60 +91,54 @@ object Target extends TargetGenerated with Applicative.Applyer[Task, Task, Resul
     )
   }
 
-  def apply[T](t: Task[T])
-              (implicit r: R[T],
-               w: W[T],
-               ctx: mill.define.Ctx): Target[T] = macro targetTaskImpl[T]
+  def apply[T](t: Task[T])(implicit r: R[T], w: W[T], ctx: mill.define.Ctx): Target[T] =
+    macro targetTaskImpl[T]
 
-  def targetTaskImpl[T: c.WeakTypeTag](c: Context)
-                                      (t: c.Expr[Task[T]])
-                                      (r: c.Expr[R[T]],
-                                       w: c.Expr[W[T]],
-                                       ctx: c.Expr[mill.define.Ctx]): c.Expr[Target[T]] = {
+  def targetTaskImpl[T: c.WeakTypeTag](c: Context)(
+    t: c.Expr[Task[T]]
+  )(r: c.Expr[R[T]], w: c.Expr[W[T]], ctx: c.Expr[mill.define.Ctx]): c.Expr[Target[T]] = {
     import c.universe._
     mill.moduledefs.Cacher.impl0[Target[T]](c)(
-      reify(
-        new TargetImpl[T](t.splice, ctx.splice, RW(w.splice.write, r.splice.read))
-      )
+      reify(new TargetImpl[T](t.splice, ctx.splice, RW(w.splice.write, r.splice.read)))
     )
   }
 
-  def source(value: Result[ammonite.ops.Path])
-            (implicit r: R[PathRef],
-             w: W[PathRef],
-             ctx: mill.define.Ctx): Input[PathRef] = macro sourceImpl
+  def source(
+    value: Result[ammonite.ops.Path]
+  )(implicit r: R[PathRef], w: W[PathRef], ctx: mill.define.Ctx): Input[PathRef] =
+    macro sourceImpl
 
-  def sourceImpl(c: Context)
-                (value: c.Expr[Result[ammonite.ops.Path]])
-                (r: c.Expr[R[PathRef]],
-                 w: c.Expr[W[PathRef]],
-                 ctx: c.Expr[mill.define.Ctx]): c.Expr[Input[PathRef]] = {
+  def sourceImpl(c: Context)(value: c.Expr[Result[ammonite.ops.Path]])(
+    r: c.Expr[R[PathRef]],
```

Comment:
@piotrkwiecinski this part still looks really odd. Any idea what's up with it?

Reply:
I'm not sure yet. I want to play around with few flags. I'll get back to you within 2 days, also I'll try to clarify other bits with guys on scalafmt gitter for better insights.

Reply:
You maybe be able to manually put some new lines in between the brackets `)(` then run scalafmt again to get a better result.